pkg/policies/opa/rego/k8s/kubernetes_service/cve_2020_8554/ensurePrivateIP.rego

```diff
@@ -0,0 +1,16 @@
+package accurics
+
+{{.prefix}}{{.name}}{{.suffix}}[service.id] {
+    service := input.{{.resource_type}}[_]
+    service.config.kind == "Service"
+    type_check(service.config.spec)
```

Comment:
IMO we should keep our checks as simple as possible. it looks like you're just trying to do the following:
service.config.spec.type == "ClusterIP"
not service.config.spec.type
not service.config.spec.externalIPs

Do we also need a check for service.config.spec.clusterIPs?

Reply:
Service type may or may not be set. If its set then I want to if its **ClusterIP**